src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -151,6 +160,16 @@ private void importFromGcs(String projectId, TableReference tableRef, @Nullable
       loadConfig.setSourceUris(gcsPaths);
       loadConfig.setDestinationTable(tableRef);
       loadConfig.setWriteDisposition(writeDisposition);
+      if (allowTimePartitioning) {
```

Comment:
Is this needed if the table already exists or does the load job always require this?

Reply:
This need only when new table should be created.